src/index.html

```diff
@@ -1,13 +1,281 @@
+            </article>
+
+            <article class="comparison__item">
+              <a href="/" class="comparison__image">
+                <img src="images/compare3.png" alt="product3">
+              </a>
+
+              <h3 class="comparison__name">Agile ride 3</h3>
+
+              <p class="comparison__descrription">The lightweight frame that has earned its street tread as a sleek, agile ride.</p>
+
+              <span class="comparison__price">$ 2 090</span>
+            </article>
+          </div>
+        </section>
+
+        <section class="main__details details" id="detail">
+          <h2 class="details__title">The Details</h2>
+
+          <div class="details__wrapper">
+            <article class="details__item">
+              <div class="details__images left-image-big">
+                <img src="images/d1.png" alt="details">
+                <img src="images/d2.png" alt="details">
+              </div>
+
+              <h3 class="details__name">Auto Unlock</h3>
+
+              <div class="details__description">
+                <p class="details__description--text">The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.</p>
+              </div>
+            </article>
+
+            <article class="details__item">
+              <div class="details__images left-image-big">
+                <img src="images/d3.png" alt="details">
+                <img src="images/d4.png" alt="details">
+              </div>
+
+              <h3 class="details__name">Range & Integrated lights</h3>
+
+              <div class="details__description">
+                <p class="details__description--text">The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+              </div>
+            </article>
+
+            <article class="details__item">
+              <div class="details__images right-image-big">
+                <img src="images/d5.png" alt="details">
+                <img src="images/d6.png" alt="details">
+              </div>
+
+              <h3 class="details__name">Hydraulic disc brakes & Lightweight</h3>
+
+              <div class="details__description">
+                <p class="details__description--text">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+              </div>
+            </article>
+
+            <button class="details__button button" onclick="window.location.href='#compare'">Explore</button>
+          </div>
+        </section>
+
+        <section class="main__contact contact" id="contact">
+          <h2 class="contact__title">Contact us</h2>
+          <div class="contact__form">
+            <form
```

Comment:
@NgineZZ 
<img width="642" alt="Monosnap Miami 2022-11-23 10-45-15" src="https://user-images.githubusercontent.com/39308717/203503898-9d13d9b4-a205-48f8-ab94-60996b217693.png">

form
- must not reload the page on submit or drop you to an error page
- could reset inputs on submit